features/networking/pod.feature

```diff
@@ -197,7 +197,8 @@ Feature: Pod related networking scenarios
   # @case_id OCP-23890
   @admin
   Scenario: A pod with or without hostnetwork cannot access the MCS port 22623 or 22624 on the master
-    Given evaluation of `env.master_hosts.first.local_ip` is stored in the :master_ip clipboard
+    Given I store the masters in the :masters clipboard
+    And the Internal IP of node "<%= cb.masters[0].name %>" is stored in the :master_ip clipboard
```

Comment:
This is useful step and can be used in future as well. We can identify INTERNAL IP of any node using this. Designed rb for this and used in such cases